### ISSUE TITLE
Information on nodes to the lower right of relationship graph tab

### ISSUE THREAD

**Issue by dlschwartz** [srophe/srophe-eXist-app#511]
On the relations visualization when I click on the circle that corresponds to a person I get information about that person on the lower right. If that could also pull the <note type="abstract">, that would be great.

**Comment by dlschwartz** [comment]
Actually, after looking around a bit, this seems to appear some of the time but not all the time. Not sure what's wrong there. Hopefully it's not a problem with the recently submitted TEI records.

**Comment by wsalesky** [comment]
Can you give me some examples of when you are seeing it and when you are not?
Just give me what you queried, and then which nodes you clicked on.

**Comment by dlschwartz** [comment]
I queried type: relationship and relationship: enmity for. I wanted to know more about Hilarian so I clicked on the node. On the lower right I got English and Syriac names along with a URI. In SBD, however, I get under the heading Identity, "Bishop of Diocaesarea in Isauria who brought charges against archdeacon Callistus on the ground that he had bought his priestly order and status." This SBD data is contained in the note type="abstract". It would be great if I could see that information in this context. Interestingly, the equivalent information under the heading Identity does appear when I click on the Severus node. I hope this explanation helps.
